scooby/knowledge.py

```diff
@@ -112,6 +111,8 @@ def in_ipykernel():
 
 def get_standard_lib_modules():
     """Return a set of the names of all modules in the standard library."""
+    import distutils.sysconfig as sysconfig
```

Comment:
Maybe add a comment like
# lazy-load for faster load time of scooby
or something along the lines?